### ISSUE TITLE
Picklist solution?

### ISSUE THREAD

**Issue by alexandroscgb** [OptimalBPM/angular-schema-form-dynamic-select#120]
Hi.

Just curious, but, is there a way to select all of the items from a multiple select and deselect them as well like in a picklist [https://www.primefaces.org/showcase/ui/data/pickList.xhtml](url)?

Thanks.

**Comment by nicklasb** [comment]
I don't think so, check the underlying components, if they have it, it is easy to add to ASFDS.
